.changeset/modern-gifts-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+"vercel": patch
```

Comment:
Is adding new functionality considered a minor change, or is it a patch because it has no impact on existing functionality?

Reply:
We usually do `minor` for new functionality.

Reply:
Updated, thanks!